examples/barotropicqgql_betaforced.jl

```diff
@@ -84,8 +85,9 @@ nothing # hide
 # numbers uniformly distributed between 0 and 1.
 random_uniform = dev==CPU() ? rand : CUDA.rand
 
-function calcF!(Fh, sol, t, clock, vars, params, grid) 
-  Fh .= sqrt.(forcing_spectrum) .* exp.(2π * im * random_uniform(eltype(grid), size(sol))) ./ sqrt(clock.dt)
+function calcF!(Fh, sol, t, clock, vars, params, grid)
+  T = eltype(grid)
+  @. Fh = sqrt(forcing_spectrum) * cis(2π * random_uniform(T)) / sqrt(clock.dt)
 
```

Comment:
This a question not review sorry. Did you try this with [`cispi`](https://docs.julialang.org/en/v1/base/math/#Base.cispi)? I am not sure if 

```julia
cispi(2 * random_uniform(T))
```

would offer any speed improvements but it says in the docs it is a more accurate method for `cis(pi*x)`. Apologies if you already looked into this!

Reply:
I did... it was same performance-wise. Docs was mentioning that `cispi` works for large `x` and since the args are O(6) then I though `cis(π` was neater than `cispi(`. :)

Reply:
Yep sounds good and as is I agree it is clearer, thanks!